pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
```

Comment:
@maximpertsov it's laudable that you're aiming at preventing common rookie errors, but it would be nice to explain *why* the `''` is included, rather than just place it there and expect users to memorize the syntax as merely part of the tool's syntax.

Just mentioning that it represents an empty query would make this more understandable (it's just the what, not the why, but at least there aren't unexplained elements to the command).

Of course, if you can think of a better way to explain the `''` in concisely the example description, we're all ears :)